hash-based-routing.html.md.erb

```diff
@@ -0,0 +1,47 @@
+---
+title: Hash-Based Routing
+owner: CF for VMs Networking
+---
+
+## <a id="purpose"></a> Purpose
+
+Hash-Based Routing is a load-balancing algorithm that distributes incoming requests to application instances based on a hash of a specific HTTP header value. This ensures consistent routing behavior where requests containing the same header value are always directed to the same instance.
```

Comment:
Can we maybe mention what typical useful headers for this scenario contain, e.g. a resource, user ID or tenant ID?